test/testcafe/framework/page-objects/components/BaseFormObject.js

```diff
@@ -124,6 +124,10 @@ export default class BaseFormObject {
     return this.el.find(FORM_INFOBOX_ERROR).innerText;
   }
 
+  getErrorBox() {
+    return Selector(CALLOUT);
```

Comment:
Does `this.getCallout(CALLOUT)` work here instead?

Reply:
Yes, that works, switched to that instead.